packages/protocol/src/utils/OrderedMsgChain.ts

```diff
@@ -74,7 +74,7 @@ class MsgChainQueue {
     peek() {
         if (this.isEmpty()) { return }
         const ref = this.queue.peek()
-        return this.pendingMsgs.get(ref)
+        return this.pendingMsgs.get(ref as MessageRef)
```

Comment:
Is this an intentional change? The variable type can be inferred automatically.

Reply:
It was intentional, 'heap' package ts signature had changed